charts/ccd-docker-user-profile-importer/templates/user-profile-importer.yaml

```diff
@@ -0,0 +1,63 @@
+{{- if .Values.importer.userprofile.enabled }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ template "hmcts.ccdupi.releaseName" . }}
+  labels:
+    app.kubernetes.io/name: {{ template "hmcts.ccdupi.releaseName" . }}
+    app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+    app.kubernetes.io/instance: {{ .Release.Name | quote }}
+    helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+  annotations:
+    helm.sh/hook: post-install,post-upgrade
+    helm.sh/hook-weight: "5"
+    helm.sh/hook-delete-policy: before-hook-creation  # Move to hook-succeeded after first charts are done 
+spec:
+  template:
+    metadata:
+      name: {{ template "hmcts.ccdupi.releaseName" . }}
+      labels:
+        app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+        app.kubernetes.io/instance: {{ .Release.Name | quote }}
+        helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    spec:
+      containers:
+        - name: {{ template "hmcts.ccdupi.releaseName" . }}
+          image: {{ .Values.importer.userprofile.image }} 
+          imagePullPolicy: IfNotPresent
+          env:
+          - name: CCD_USER_PROFILE_URL
+            value: http://{{ .Release.Name }}-user-profile-api
```

Comment:
this should be externally configurable like .Values.useprofileUrl may be with tpl enabled .. doesn't make sense to be releasename dependent

Reply:
fixed